sections/semantics-forms.include

```diff
@@ -1209,7 +1209,7 @@
   <pre highlight="html">
 &lt;form&gt;
   &lt;table&gt;
-    &lt;caption&gt;Example &lt;label&gt;'s for attribute with &lt;label&gt;&lt;/caption&gt;
+    &lt;caption&gt;Example, &lt;label&gt;'s for attribute&lt;/caption&gt;
```

Comment:
@stevefaulkner I think @chaals 's comment was to add a colon, but a comma was added instead... (unless the plan changed as you're in the f2f meeting together?). (btw I hope it is ok for me to comment this way).